support-frontend/assets/helpers/abTests/abtestDefinitions.js

```diff
@@ -103,10 +103,10 @@ export const tests: Tests = {
       },
     },
     isActive: true,
-    referrerControlled: false,
+    referrerControlled: true,
     targetPage: pageUrlRegexes.subscriptions.digiSub.nonGiftLandingNotAusNotUS,
```

Comment:
I believe we can remove this optional `targetPage` prop in the test definition. It's not needed on `referrerControlled` ab tests as the query string parameters will determine whether a user is in the test.